### ISSUE TITLE
ssh error "This bastion supports interactive sessions only. Do not supply a command"

### ISSUE THREAD

**Issue by vickydev90** [warpdotdev/Warp#3264]
### Operating System

MacOS

### Operating System and Version

Ventura 13.1

### Local Shell Version

5.8.1

### Remote Shell Version

4.2.46(2)-release

### Warp Version

v0.2023.06.13.08.03.stable_03

### I have tried commenting out my system & user rcfiles and warp still will not bootstrap

- [X] Yes

### Screenshots

while trying ssh to bastion ,i am getting error `This bastion supports interactive sessions only. Do not supply a command` .
when trying from normal terminal , it works without any issue.

<img width="971" alt="image" src="https://github.com/warpdotdev/Warp/assets/23333584/a9069e57-e05a-44c8-8d5a-b4ad108636a7">

### Include shell xtrace output

<img width="962" alt="image" src="https://github.com/warpdotdev/Warp/assets/23333584/1f83bcd4-dbb5-4537-9ea4-c68b489068ce">

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore) - linear-label:e7dfaa84-5fdb-4a00-b754-d8912da923fa

None

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by zachbai** [comment]
Hey @vickydev90 -- this is an issue with Warp's SSH wrapper not being compatible with SSH proxies or jumpservers.

An update to the wrapper that would fix this issue isn't likely to get done any time soon; its actually a pretty tough problem to solve.

On the other hand, you can use Warp's [subshells feature](https://docs.warp.dev/features/subshells) to circumvent the wrapper while still getting basically the same experience. 

If you add `ssh bat` as a "subshell compatible command" as described [here](https://docs.warp.dev/features/subshells#adding-compatible-commands), the next time you run `ssh bat`:
1) Select your destination host/server (don't click "Warpify subshell" until you're dropped into a session on that dest host)
2) 'Warpify' the session via the button on the banner

![image](https://github.com/warpdotdev/Warp/assets/10970845/d4f3f4f1-8dec-46b2-a6d9-a9016e2f748c)

**Comment by dannyneira** [comment]
@vickydev90 Any luck with the above steps? https://github.com/warpdotdev/Warp/issues/3264#issuecomment-1609990646

**Comment by danilgotvyansky** [comment]
No, the solution above doesn't work. Same error and it even doesn't let to connect, so you are not able to click Warpify shell after the successful connection
